recipes/pyqt5-sip/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyqt5-sip" %}
+{% set version = "12.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyQt5_sip-{{ version }}.tar.gz
```
this package is a bit unusual, I will take a look on that later to improve grayskull